src/OpenTelemetry.Instrumentation.AWS/OpenTelemetry.Instrumentation.AWS.csproj

```diff
@@ -9,9 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.5.1.24" />
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.5.0.27" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.5.0.26" />
+    <PackageReference Include="AWSSDK.Core" Version="[3.5.1.24,4.0)" />
```

Comment:
without this change also, these versions could have been used right? The title of the PR is misleading, as this is not adding support for newer versions, instead putting upper bound on the versions supported?

Reply:
Correct, they can be used, but the fixed dependance on the outdated versions causes warnings in all of our projects so my intent was to open up the accepted versions to the current 3.7.x versions and above. I'm struggling a bit with both the package boundary notation and pr/codespace/traveling so please feel free to ditch this whole thing if it seems untenable. Just hoping to use 3.7 without endless notifications about 3.5.

Reply:
For that goal, this PR should bump the minimum supported version from 3.5 to 3.7. I'll let the owners of this package decide if they are okay with that change.

Reply:
Sorry, hopelesly lost in codespaces/pr/screen glare while traveling. Would it be possible to open up the accepted range of sdk's to allow current ones (thereby letting the end user decide which version they'll use) rather than moving to the current version and fixing it there? Moving to 3.7 will resolve my issue but I worry that someone stuck on 3.5 will lose their access to updates. Could be I'm out of my depth on how those ranges affect things.